src/components/Common/LanguagePicker.js

```diff
@@ -123,7 +135,25 @@ export const LanguagePicker = ({
         )}
       />
 
-      <Image source={selectLanguageImage} style={styles.image} />
+      {languageCode !== 'en-US' && languageCode !== 'ja-JP' && (
+        <View style={styles.ackBlock}>
+          {intl.formatMessage(messages.contributors) !== '_' ? (
```

Comment:
I think this is wrong. You use `_` here but `-` as the default string.

Reply:
yes, last minute error when making the screenshots